fibonacci/main.go

```diff
@@ -3,12 +3,29 @@ package main
 import "fmt"
 
 func main() {
-	fmt.Println(fibonacci(7))
+	const n = 5
+	fmt.Println("Calculating %d sequence\n", n)
 }
 
-func fibonacci(n int) int {
-	// Fill me in!
-	// When n is 0 or 1, fibonacci(n) should just be n.
-	// When n is any larger number,
-	// fibonacci(n) should be the sum of the previous two fibonacci numbers.
+func fibonacci(n int, old, new, calc string) int {
+
+	if n == 0 {
+		return
+	}
+
+	fibonacci(n-1, old, calc, new)
+
+	fmt.Printf("  n  ", old, new)
+
+	fibonacci(n-1, calc, new, old)
+
 }
```

Comment:
You probably got an error on this line too saying you reached the end of the function without returning a value.